ktlint-ruleset-experimental/src/test/kotlin/com/pinterest/ktlint/ruleset/experimental/NoFirstLineBlankInMethodBlockRuleTest.kt

```diff
@@ -0,0 +1,111 @@
+package com.pinterest.ktlint.ruleset.experimental
+
+import com.pinterest.ktlint.core.LintError
+import com.pinterest.ktlint.test.format
+import com.pinterest.ktlint.test.lint
+import org.assertj.core.api.Assertions.assertThat
+import org.testng.annotations.Test
```

Comment:
This is the wrong import. You need to use `import org.junit.Test`.

Reply:
All the other tests use this import `import org.testng.annotations.Test` though

Reply:
The do not anymore. Can you please rebase with master?
`git rebase -i origin/master`

Reply:
Ah ! Got it.. Will fix it..